strings.txt

```diff
@@ -20797,3 +20797,63 @@
     zh-Hant = 叫計程車服務暫時無法使用。
     el = Η παραγγελία ταξί είναι προσωρινά μη διαθέσιμη.
     sk = Objednanie taxíka je dočasne nedostupné.
+
+  [whatsnew_traffic]
+    comment = december
+    en = Traffic Data in 36 Countries
+    ru = Пробки в 36 странах мира
+
+  [whatsnew_traffic_text]
+    comment = december
+    en = Up-to-date traffic info and fastest driving routes based on it!
+    ru = Актуальная информация о пробках на дорогах и самые быстрые маршруты с их учетом в 36 странах мира!
+
+  [whatsnew_traffic_roaming]
+    comment = december
+    en = Traffic Info in Roaming
+    ru = Пробки даже в роуминге
+
+  [whatsnew_traffic_roaming_text]
+    comment = december
+    en = Requires less than 1 MB of mobile bandwidth per trip.
+    ru = Требует менее 1 МБ мобильного трафика за поездку.
+
+  [whatsnew_order_taxi]
+    comment = december
+    en = Order A Taxi
+    ru = Вызов такси
+
+  [whatsnew_order_taxi_text]
+    comment = december
+    en = Order a taxi button on most location pages.
```

Comment:
Taxi order button on most location pages.
A button to order taxi on most location pages.
Most of the location pages now contain a taxi order button.
Taxi can be ordered from most location pages.
Book a taxi from location page.